publisher-provider/publisher-provider-app/src/main/java/nl/idgis/publisher/provider/database/OracleDatabaseCursor.java

```diff
@@ -3,21 +3,14 @@
 import java.sql.Blob;
 import java.sql.Clob;
 import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
 
+import nl.idgis.publisher.provider.database.messages.AbstractDatabaseColumnInfo;

```

Comment:
(cosmetisch) import staat op op verkeerde plaats. java.*, org.* en nl.* imports staan normaalgesprokken in afzondelijke blokken.

Reply:
Opgelost